Microsoft.Toolkit.Uwp.UI.Controls/AdaptiveGridView/AdaptiveGridView.cs

```diff
@@ -100,6 +100,13 @@ protected override void PrepareContainerForItemOverride(DependencyObject obj, ob
         /// <returns>The calculated item width.</returns>
         protected virtual double CalculateItemWidth(double containerWidth)
         {
+            // quick check for containerWidth to avoid cyclic cases where size
+            // of the container is changed by the size of the items (availableSize == Inf)
+            if (containerWidth > Window.Current.Bounds.Width * 2)
```

Comment:
What if the containerWidth is actually greater than  Window.Current.Bounds.Width * 2 ?